### PR TITLE
add a page on System Security Plans (SSPs)

### PR DESCRIPTION
**Pull request by afeld**
/cc @stvnrlly @DavidEBest @geramirez

### CODE REVIEW

File: _pages/ato/ssp.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+### Template
+
+Add the following to your repository, with the information in the `<angle brackets>` filled in. Note that there are `TODO`s in some places where we need more documentation – for now, reach out to Noah Kunin for help with what should be filled in there.
+
+```yaml
+---
+name: <project name>
+# this should match the infrastructure Tag
+uniqueID: <partner-acronym>-<beginning-of-agreement>-<end-of-agreement>-<iaa-number>
+version: 0.1.0
```
Comment (afeld) [inline _pages/ato/ssp.md]: Is this the application version, or the `system-security-plan.yml` schema version?

File: _pages/ato/ssp.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+phase: <discovery|alpha|beta|live>
+information-types:
+- <TODO>
```
Comment (afeld) [inline _pages/ato/ssp.md]: Anyone know what should go here?

File: _pages/ato/ssp.md (hunk 1 above)
Comment (DavidEBest) [inline _pages/ato/ssp.md]: There isn't consistency between existing examples (ATF = 0.1.0, EKIP = 1.0.0, FEC = 1.2.0), so I expect this is related to the application's version.

File: _pages/ato/ssp.md (hunk 2 above)
Comment (DavidEBest) [inline _pages/ato/ssp.md]: That comes from NIST SP800-60 (http://csrc.nist.gov/publications/nistpubs/800-60-rev1/SP800-60_Vol1-Rev1.pdf). 

`D26 Civilian Operations` is what I see in most of our SSPs. D.26 is 'Direct Services for Civilians'.

File: _pages/ato/ssp.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+### Template
+
+Add the following to your repository as `system-security-plan.yml`, with the information in the `<angle brackets>` filled in:
+
+```yaml
+---
+name: <project name>
+uniqueID: <MB number – see https://docs.google.com/spreadsheets/d/1hjCYIskgD_x_MI1ehXoiz2Qvsyxj1yK3fxabkezMPiE/edit#gid=0>
+version: <application version>
```
Comment (afeld) [inline _pages/ato/ssp.md]: @NoahKunin What should this value be? Not sure that I've seen any of our applications be versioned explicitly.

**Comment by afeld** [comment]
Updated!

File: _pages/ato/ssp.md (hunk 3 above)
Comment (DavidEBest) [inline _pages/ato/ssp.md]: @NoahKunin Any context for this value? Was a it determined by someone on the project team?

**Pull request merged**